Fix basic checks

C0rWin opened the pull request:
A few recent PRs broke basic CI checks, this commits address linter and docs update issues.

Signed-off-by: Artem Barger <artem@bargr.net>

C0rWin commented:
I do not understand this


 ```go
  --- docs/source/commands/ledgerutil.md	2023-01-13 12:00:11.990578077 +0000
+++ /tmp/ledgerutil_preamble.md.vikqb	2023-01-13 12:04:14.292436188 +0000
@@ -144,8 +144,8 @@
   -f, --firstDiffs=10        Maximum number of differences to record in
                              first_diffs_by_height.json. Requesting a report
                              with many differences may result in a large amount
-                             of memory usage. Defaults to 10. If set to 0,
-                             will not produce first_diffs_by_height.json.
+                             of memory usage. Defaults to 10. If set to 0, will
+                             not produce first_diffs_by_height.json.
 
 Args:
   <snapshotPath1>  First ledger snapshot directory.
@@ -169,8 +169,8 @@
   <snapshotDiffsPath>  Path to json file containing list of target records to
                        search for in transactions. This is typically output from
make: *** [Makefile:121: check-help-docs] Error 2
                        ledgerutil compare.
-  [<blockStorePath>]   Path to file system of target peer, used to access
-                       block store. Defaults to '/var/hyperledger/production'.
+  [<blockStorePath>]   Path to file system of target peer, used to access block
+                       store. Defaults to '/var/hyperledger/production'.
                        IMPORTANT: If the configuration for target peer's file
                        system path was changed, the new path MUST be provided.
 ```
The command line help docs are out of date and need to be regenerated
Error: Process completed with exit code 2.
```

Running locally, I do not see these problems.

denyeart commented:
@C0rWin I'm not seeing any issues on latest main, opened a test PR - https://github.com/hyperledger/fabric/pull/3921

C0rWin commented:
> @C0rWin I'm not seeing any issues on latest main, opened a test PR - #3921

Well, this is very strange, I do really see why I'm keep getting this errors 🤔 

PS. Suggested this PR cause I started to see errors here: https://github.com/hyperledger/fabric/pull/3902

C0rWin committed:
Fix basic checks

Few recent PRs broke basic CI checks, this commits address
linter and docs update issues

Signed-off-by: Artem Barger <artem@bargr.net>

denyeart commented on core/ledger/kvledger/hashcheck_pvtdata.go:
```diff
@@ -25,7 +25,9 @@ import (
 // to remove the key-values that were not present in the snapshot (most likely because, they
+//
 //	were over-written in a later version). Also, the keys that have been purged explicitly by a
+//
```
I think you just need to remove the indent here, rather than add new lines.

C0rWin replied:
it's been added with goimports, none of these changes was done manually.

denyeart replied:
I understand, I'm suggesting goimports was confused due to the indent, it would be better to hand edit this one in a more expected way.

denyeart commented:
@Mergifyio backport release-2.5

denyeart commented:
Thanks @C0rWin , the auto-backport had some conflicts, would you be able to create a release-2.5 PR for this as well?

C0rWin commented:
> Thanks @C0rWin , the auto-backport had some conflicts, would you be able to create a release-2.5 PR for this as well?

yes sure
